include/RB/vx_rb_ref_data.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023 ETAS Ltd.
```

Comment:
Can this license be changed to the Khronos license?

Reply:
done, change to khronos, file's name also change to remove the rb part. It is now called "tivx_supp_ref_data.h"